src/components/scenes/Staking/StakeOptionsScene.tsx

```diff
@@ -96,28 +84,12 @@ export const StakeOptionsScene = (props: Props) => {
     )
   }
 
-  const sceneHeader = useMemo(
-    () => (
-      <SceneHeader style={styles.sceneHeader} title={sprintf(s.strings.staking_change_add_header, currencyCode)} underline withTopMargin>
-        {icon}
-      </SceneHeader>
-    ),
-    [currencyCode, icon, styles.sceneHeader]
-  )
-
-  if (stakePolicies.length === 0)
-    return (
-      <SceneWrapper background="theme">
-        <FillLoader />
-      </SceneWrapper>
-    )
-
```

Comment:
This looks like it shouldn't have been removed. I added fixup commits for this and for the next commit.